terraform/shared/shared.tf.json.template.py

```diff
@@ -44,7 +44,11 @@ def conformance_pack(name: str) -> str:
     CloudTrailAlarm(name='api_unauthorized',
                     statistic='Sum',
                     filter_pattern='{($.errorCode="*UnauthorizedOperation") || ($.errorCode="AccessDenied*")}',
-                    threshold=12,
+                    # We anticipate two AWS Config related alarms daily, with
+                    # the specified delivery_frequency below, and two SSM alarms
+                    # weekly or upon a GitLab instance reboot. Plus a margin of
+                    # four alarms to account for misc. azul-devs activity.
```

Comment:
```suggestion
                    # We expect two false alarms caused by AWS Config every day (see delivery_frequency below), two false alarms per week from the SSM Agent running on the GitLab instance
                    #, and two additional false alarms from the agent for every reboot of the GitLab instance, which we expect to occur at most once per day. We leave a margin of
                    # of two false alarms to account for [what exactly].
```

"miscellaneous azul-devs activity" is hand-waving. We've observed that the above number of expected false alarms fluctuates, or that our notion of "day" doesn't sync up with AWS'. I believe that's why we added the margin.